Source/Charts/Components/ChartLimitLine.swift

```diff
@@ -68,18 +68,7 @@ open class ChartLimitLine: ComponentBase
         }
         set
         {
-            if newValue < 0.2
-            {
-                _lineWidth = 0.2
-            }
-            else if newValue > 12.0
-            {
-                _lineWidth = 12.0
-            }
-            else
-            {
-                _lineWidth = newValue
-            }
+            _lineWidth = newValue.clamped(to: 0.2...12)
```

Comment:
here 12 is cast to Double by default? is this new swift feature?

Reply:
This has been the case since Swift 1